doc/api/esm.md

```diff
@@ -1461,10 +1469,10 @@ export function resolve(specifier, context, defaultResolve) {
   }
 
   // Let Node.js handle all other specifiers.
-  return defaultResolve(specifier, context, defaultResolve);
+  return null;
 }
 
-export function getFormat(url, context, defaultGetFormat) {
+export function getFormat(url) {
```

Comment:
The previous examples always had the full signatures, even if not all arguments were used. I think that helps educate people.